EditPortfolio.js

```diff
@@ -0,0 +1,37 @@
+const addExperienceBtn = document.getElementById("addExperienceBtn");
+const experienceForm = document.getElementById("experienceForm");
+const experienceList = document.getElementById("experienceList");
+const experienceEntryForm = document.getElementById("experienceEntryForm");
+
+addExperienceBtn.addEventListener("click", () => {
+  experienceForm.style.display = "block";
+});
+
+experienceEntryForm.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const companyName = document.getElementById("companyName").value;
+  const startDate = document.getElementById("startDate").value;
+  const endDate = document.getElementById("endDate").value;
+  const description = document.getElementById("description").value;
```

Comment:
We can extract this outside of the function and use it in multiple places to avoid duplication.

Reply:
updated